iphone/cli/hooks/frameworks.js

```diff
@@ -201,7 +210,7 @@ class FrameworkManager {
 		let simulatorArchitectures = new Set([ 'i386', 'x86_64' ]);
 
 		for (let frameworkInfo of this._frameworks.values()) {
-			if (frameworkInfo.type !== FRAMEWORK_TYPE_DYNAMIC) {
+			if (frameworkInfo.type !== FRAMEWORK_TYPE_DYNAMIC || frameworkInfo.wrapperName.endsWith('xcframework')) {
```

Comment:
we should probably just extract a property/method on FrameworkInfo that tells you if it's an xcframework. We have conditionals like this all over the code now. (In fact, design-wise it may make sense to have a subclass called `XCFrameworkInfo` that handles those small changes as method/getter overrides)